crates/bitwarden/src/tool/exporters/mod.rs

```diff
@@ -0,0 +1,33 @@
+use schemars::JsonSchema;
+
+use crate::{
+    error::Result,
+    vault::{CipherView, CollectionView, FolderView},
+};
+
+mod client_exporter;
+
+#[derive(JsonSchema)]
+#[cfg_attr(feature = "mobile", derive(uniffi::Enum))]
+pub enum ExportFormat {
+    Csv,
+    Json,
+    AccountEncryptedJson, // TODO: Should we deprecate this option completely?
```

Comment:
I don't think this format is particularly useful (and mostly replaced by the password encrypted JSON) and will just cause issues when users try to import it into a new account or after rotating keys, so I'm in support of deprecating it.